middle_end/flambda2/terms/code_size.ml

```diff
@@ -363,6 +363,7 @@ let binary_prim_size prim =
   match (prim : Flambda_primitive.binary_primitive) with
   | Block_load (kind, _) -> block_load kind
   | Array_load (kind, _mut) -> array_load kind
+  | Array_vector_load (_vec, kind, _mut) -> array_load kind
```

Comment:
not sure it should be the same size as an array load, I'd think that the `128`-bits load instructions take more space than regular word-sized ones ?

Reply:
The encoding of `movupd` (m128<->xmm) is the same size as `movq` (m64<->xmm)

They're actually both a bit larger than `mov` on integer regs, but it doesn't look like code_size represents size at that granularity

Reply:
Ok, i'm good with that explanation, :)